vowpalwabbit/parse_args.cc

```diff
@@ -1048,6 +1048,12 @@ void parse_example_tweaks(options_i& options, VW::workspace& all)
   }
 
   all.loss = getLossFunction(all, loss_function, loss_parameter);
+  if (options.was_supplied("quantile_tau") && all.loss->getType() != "quantile")
```

Comment:
why would this happen though?

Reply:
and why only test for quantile_tau, I'm assuming this can happen with any other loss related arg that is later overridden somehow

Reply:
quantile is the only loss function which accepts an argument.